app/src/main/java/com/ballboycorp/com/topik/SonarBugSample.kt

```diff
@@ -0,0 +1,19 @@
+package com.ballboycorp.com.topik
+
+interface Sample {
+    fun method()
+
+    fun anotherMethod()
+}
+
+class SampleImpl : Sample {
+    override fun method() {
+
+    }
+
+    override fun anotherMethod() {
+        var some = 1
+        some = some
```

Comment:
Remove or correct this useless self-assignment. [author:musooff@gmail.com, component:musooff_Topik_AYB-ZFlS8-xhCYxQvxYf:app/src/main/java/com/ballboycorp/com/topik/SonarBugSample.kt, creationDate:2022-05-01T16:56:35+0900, debt:3min, effort:3min, flows:[], hash:eb25a3d25b2cf17a7d5f8bb217a5ab0a, key:AYB-q2G40n16_HoDUh6w, line:16, message:Remove or correct this useless self-assignment., project:musooff_Topik_AYB-ZFlS8-xhCYxQvxYf, quickFixAvailable:false, rule:kotlin:S1656, scope:MAIN, severity:MAJOR, status:OPEN, tags:[], textRange:[endLine:16, endOffset:19, startLine:16, startOffset:8], type:BUG, updateDate:2022-05-02T16:47:05+0900]